docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
```

Comment:
if we go for only one official Docker per GE, maybe we are not going to need a docker folder and we might put the Dockerfile at the root of the repo, WDYT?